CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 CHANGELOG
 =========
 
+Upgrade to Django 1.9
+---------------------
+* Tests were changed to call `save` on some test objects due to `new behavior for related objects <https://docs.djangoproject.com/en/1.9/releases/1.9/#bulk-behavior-of-add-method-of-related-managers>`_.
+* Updated url files to make them plain lists. `django.conf.urls.patterns()` `will be removed in 1.10 <https://docs.djangoproject.com/en/1.9/ref/urls/>`_.
+* `SortedDict` `was moved <https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.datastructures.SortedDict>`_ from `django.utils.datastructures.SortedDict` to `collections.OrderedDict`.
+* Template settings were changed to `the new template api <https://docs.djangoproject.com/en/1.9/releases/1.8/#multiple-template-engines>`_.
+
+There are still two depreciation warnings for 1.10. `SubfieldBase` will be removed in 1.10. It is called by `MultiSelectField` in `People.Models` and by `django_date_extensions` in `Jounrals.Models`.
+
```

Comment:
These detailed change logs with links are super helpful!  Thanks for taking the time to do that, I'll try to remember doing that myself in future.

Minor typo on `Jounrals.Models`